### PR TITLE
Add styles for CRUD example

### PR DESCRIPTION
**Pull request by scabbiaza**
Update webpack config in CRUD example: keep configs only for FE
Note: `new ExtractTextPlugin` in plugins makes merging problematic

### CODE REVIEW

**Comment by scabbiaza** [comment]
Decided to separate filters and sorts by the linebreak, otherwise `users` list didn't look good.

screen shot 2017-11-29 at 19 24 43

screen shot 2017-11-29 at 19 24 50

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,4 @@
 node_modules
 npm-debug.log*
 */*/public/js
-*/*/public/css
+examples/7.crud/public
```
Comment (ivan-kleshnin) [inline .gitignore]: Why?

File: examples/7.crud/client/home/PostIndex.js [hunk 2]
```diff
@@ -3,9 +3,13 @@ import * as R from "ramda"
 import React from "react"
 import PostItem from "./PostItem"
 
+// in order to Webpack move this file to `public` folder
```
Comment (ivan-kleshnin) [inline examples/7.crud/client/home/PostIndex.js:6]: Also to minimize and possibly inject them as base-64 (for images).

File: examples/7.crud/webpack.config.js [hunk 3]
```diff
@@ -43,7 +51,7 @@ let configs = {
     ],
   },
   plugins: [
-    new ExtractTextPlugin("css/[name].css"),
+    new ExtractTextPlugin("[name].css"),
```
Comment (ivan-kleshnin) [inline examples/7.crud/webpack.config.js]: Why `/css/` and `/js/` suffixes are removed?

**Comment by ivan-kleshnin** [comment]
Great, thanks!

**Pull request merged**